packages/components/tabs/src/tab-group.ts

```diff
@@ -484,7 +500,9 @@ export class TabGroup extends ScopedElementsMixin(LitElement) {
       this.menuItems = undefined;
     }
 
-    this.selectedTab?.scrollIntoView(false);
+    if (this.selectedTab) {
+      this.#scrollIntoViewIfNeeded(this.selectedTab, 'instant');
```

Comment:
it looks like it's not working? Eg. on the website.

Reply:
Marking as resolved. The situation on the website is murky because we're using links in the tabs there, which causes the page to reload.